.orchestra/config/components/revng-vscode-web.yml

```diff
@@ -0,0 +1,71 @@
+#@ load("@ytt:data", "data")
+#@ load("@ytt:overlay", "overlay")
+#@ load("/lib/create_component.lib.yml", "single_build_component")
+
+#@yaml/text-templated-strings
+---
+#@ def revng_vscode_web_args():
+repository: vscode
+binary_archives: private
+dependencies:
+- revng
```

Comment:
```suggestion
- revng-c
```

Reply:
Depends on our intentions.
`revng` is *required* because we install the `launch-vscode-web` subcommand
I guess you want `revng-c` so that the daemon can be launched, but I don't know if it's ok to mandate this strong coupling

Reply:
Ok, changed to `revng-c`